### DISCUSSION TITLE
Upgrade to a new product after placing an order.

### DISCUSSION THREAD

**Discussion by ramkishore-yerubandi** [vendurehq/vendure#2618] [Q&A]
We would like to provide administrators with the capability to change the product variant on an order line after a customer has placed an order. The purpose is to offer a complimentary upgrade of the product to specific customers. Could you please suggest a method to achieve this? I've observed that updating the productVariantId on the order line through a custom mutation is possible. However, I am uncertain whether this approach accurately reflects the stock levels. Additionally, I would appreciate suggestions on a more effective implementation.

**Comment by michaelbromley** [comment]
Hi,
Just updating the ID to point to a different variant will cause issues as you have predicted, as stock is not accounted for. You can however modify the order via the Admin UI (use the context menu on the order detail page).

**Reply by ramkishore-yerubandi** [reply]
i am using vendure 1.9.7. I think Modify Order is only there on Latest

**Reply by michaelbromley** [reply]
Order modification has been around for a while I believe, and should be in that version. Can you check the available menu options in the order detail page?
